src/renderer/getDiForUnitTesting.tsx

```diff
@@ -111,6 +112,9 @@ export const getDiForUnitTesting = (opts: { doGeneralOverrides?: boolean } = {})
 
     di.override(historyInjectable, () => createMemoryHistory());
     di.override(legacyOnChannelListenInjectable, () => () => noop);
+
+    di.override(storageSaveDelayInjectable, () => 0);
```

Comment:
Why is this override different than that of `main`'s `getDiForUnitTesting`?

Reply:
This is used only by `createStorage` which is `renderer` thing.

Reply:
Then why should `main` even know about it?

Reply:
It doesn't?